sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1325,7 +1333,8 @@ def __init__(
       ignore_insert_ids=False,
       with_batched_input=False,
       ignore_unknown_columns=False,
-      max_retries=MAX_INSERT_RETRIES):
+      max_retries=MAX_INSERT_RETRIES,
+      max_insert_payload_size=MAX_INSERT_PAYLOAD_SIZE):
```

Comment:
why can max_insert_payload_size be changed since https://cloud.google.com/bigquery/quotas#streaming_inserts is always there?

Reply:
In case users want to control the batch size of their inserts (e.g. to have smaller more frequent batch inserts). I'll add a check that this doesn't exceed the true maximum

Reply:
Done